build/nuget-properties.props

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)\common.props" />
+
+  <PropertyGroup>
+    <!--
+    If you try to override output path directly from the command line (or release.yml) then
+    you lose the target-framework subdirectories (for example, instead of building into $(OutputPath)\netstandard2.0
+    you just build into $(OutputPath)) - this usually isn't what you want. The workaround here
+    is to set an intermediate variable BaseOutputPath from the command line (or release.yml) and
+    then set that to OutputPath here, which preserves the target-framework subdirectories.
+    -->
+    <OutputPath>$(BaseOutputPath)</OutputPath>
+    <Authors>Microsoft</Authors>
+    <Description>Infer.NET is a framework for running Bayesian inference in graphical models. It can also be used for probabilistic programming. This package contains the Infer.NET Compiler, which takes model descriptions written using the Infer.NET API and converts them into inference code.</Description>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <ProjectUrl>https://dotnet.github.io/infer</ProjectUrl>
```

Comment:
nuget.org does not seem to recognize this attribute.  It says "Project URL: (none specified)".

Reply:
Fixed.